axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveValidateService.java

```diff
@@ -242,6 +242,11 @@ public void validate(Move move, boolean updateCustomerAccount) throws AxelorExce
 
     this.updateValidateStatus(move, dayBookMode);
 
+    move.setCodeJournal(move.getJournal().getCode());
+    move.setNameJournal(move.getJournal().getName());
+    move.setCodeCompany(move.getCompany().getCode());
+    move.setNameCompany(move.getCompany().getName());
+
```

Comment:
Please, move this code statement in the completeMoveLines(Move move). You can also rename it into completeMoveStaticFields(Move move)

https://github.com/axelor/axelor-open-suite/blob/da95f619e1f90381b96cd5ed898beef10c142e5a/axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveValidateService.java#L90